core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -505,6 +522,13 @@ public boolean getShared() {
     public long getMountId() {
       return mMountId;
     }
+
+    /**
+     * @return the ufsClient corresponding to this ufs
+     */
+    public UfsManager.UfsClient getUfsClient() {
```

Comment:
is this used?

Reply:
Yes. It is used in `loadMountPointDirectoryMetadata `